definitions-terms.md

```diff
@@ -36,7 +36,7 @@ OCI Image is a specific type of artifact. However, an OCI image is not meant to
 
 ## Registry
 
-A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
+A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers](implementers.md) for a list of registries that support OCI Artifacts.
```

Comment:
believe it or not the typical pattern for linking to the bottom of the md is [][] 

though linking directly the file with () also works might as well use the redirection to below

```suggestion
A registry, or container registry, is an instance of the [distribution-spec]. See [Implementers][implementers] for a list of registries that support OCI Artifacts.
```

Reply:
@Zhurik, we appreciate the effort for the PR. If the additional tweaks are too much detail, we can merge this as is, and tweak a bit further as the link works with your PR. We just have a dangling entry in the [links section at the bottom](https://github.com/opencontainers/artifacts/pull/64/files#diff-da55140985df1701b1b0ece67f61bfde0bc2e2e85aeea1cb3d9bba978945b1deL51) of the file.